WcaOnRails/config/initializers/translations.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+if defined?(Rails::Server) # See: https://stackoverflow.com/a/44441168
+  Rails.configuration.after_initialize do
+    modification_timestamp = Timestamp.find_or_create_by!(name: 'en_translation_modification')
+    latest_modification_date = DateTime.parse(`git log -1 --format='%ai' #{Rails.root.to_s}/config/locales/en.yml`)
+    if modification_timestamp.date != latest_modification_date
```

Comment:
I think it might be good to check if the timestamp is newer than the one in the database, not just different. Here's why:

A few times when things have been very broken on production, we've skipped the normal PR flow and run git checkout and deploy directly on production. This has been useful to quickly rollback a change, but I think might cause this email to be repeatedly sent.

Similarly, should we be checking if we're on production here? What happens when people deploy changes to staging?